src/components/project/list.vue

```diff
@@ -81,12 +134,24 @@ export default {
 </script>
 
 <style lang="scss">
-#project-list table { table-layout: fixed; }
+@import '../../assets/scss/mixins';
+
+#archived-projects {
+  .project-title {
+    a { @include text-link; }
+    color: $color-action-foreground;
```

Comment:
Similar question as above, do we need to specify `$color-action-foreground` here?